frontend/javascripts/statusbar.js

```diff
@@ -0,0 +1,177 @@
+// @flow
```

Comment:
I think this file should be moved to `webknossos/frontend/javascripts/oxalis/view`.